src/index.ts

```diff
@@ -1,7 +1,11 @@
 export {
     PartialConfig as AwsRumConfig,
-    Orchestration as AwsRum
+    Orchestration as AwsRum,
+    PartialCookieAttributes as PartialCookieAttributes,
+    PageIdFormat as PageIdFormat,
+    Telemetry as Telemetry
```

Comment:
nit: We don't need to add an alias for names we aren't changing.

Reply:
Makes sense!